index.bs

```diff
@@ -155,12 +157,15 @@ interface SpeechRecognition : EventTarget {
     attribute boolean continuous;
     attribute boolean interimResults;
     attribute unsigned long maxAlternatives;
+    attribute SpeechRecognitionMode mode;
 
     // methods to drive the speech interaction
     undefined start();
     undefined start(MediaStreamTrack audioTrack);
     undefined stop();
     undefined abort();
+    boolean onDeviceWebSpeechAvailable(DOMString lang);
```

Comment:
At a first glance, this property looks like an event due to the "on" prefix.

A way to avoid that misinterpretation would be to prepend another prefix to the name, e.g. `isOnDeviceWebSpeechAvailable`.

P.S. This is random feedback that does not carry any weight. I just read this proposal out of interest and noticed this.